src/azure-cli/azure/cli/command_modules/search/custom.py

```diff
@@ -166,9 +166,9 @@ def update_search_service(instance, partition_count=0, replica_count=0, public_n
 
     replica_count = int(replica_count)
     partition_count = int(partition_count)
-    if replica_count > 0:
+    if replica_count > 0:  # pylint: disable=consider-using-min-builtin
         instance.replica_count = replica_count
```

Comment:
`disable=consider-using-min-builtin` should not be raised.
See https://github.com/pylint-dev/pylint/issues/9800